main.js

```diff
@@ -17,16 +17,39 @@ var maxRange = document.querySelector('#max-range');
 var minSpan = document.querySelector('#min-span');
 var maxSpan = document.querySelector('#max-span');
 var updateButton = document.querySelector('#update-button');
+var errorIcon = document.querySelector('#error');
+var winnerName = document.querySelector('#winner-name');
+var closeWinnerOutputButton = document.querySelector('#close-winner-output');
+var winnerOutput = document.querySelector('.winner-output');
+var winnerOutputSection = document.querySelector('.container2');
 
 submitButton.disabled = true;
 clearButton.disabled = true;
 resetButton.disabled = true;
+updateButton.disabled = true;
 
 document.addEventListener('keyup', enableSubmit);
 document.addEventListener('keyup', enableClear);
+document.addEventListener('keyup', checkRange);
 clearButton.addEventListener('click', clearInputs);
 submitButton.addEventListener('click', updateLatestGuess);
 updateButton.addEventListener('click', updateRange);
+resetButton.addEventListener('click', resetForm);
+closeWinnerOutputButton.addEventListener('click', removeWinnerOutput);
+
+function removeWinnerOutput() {
+  winnerOutput.remove();
+}
+
+function resetForm() {
+  minSpan.innerText = "?";
+  maxSpan.innerText = "?";
+  oneName.value = "";
+  twoName.value = "";
+  minRange.value = "";
+  maxRange.value = "";
+
```

Comment:
keep an eye on unnecessary whitespace like this

Reply:
Thanks.  I didn't remove any whitespace because I know that we will clean up in the future.  I'll try to keep a habit of cleaning up whitespaced will I code.